strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/providers/storage/FileSystemStorageProvider.java

```diff
@@ -51,62 +49,7 @@ public void register()
     }
 
     @Override
-    public OutputStream getOutputStreamImplementation(Path artifactPath)
```

Comment:
In line with my earlier comments in this pull regarding the removal of the `get[Input|Output]Stream`, I'd like to hear your thoughts on this part here as well.